pkg/services/object/util/key.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 var (
-	// todo(alexvanin): should be a part of status API
+	// TODO(alexvanin): #1144 should be a part of status API
```

Comment:
Link to the neofs-api issue. Personally, I would delete it, because issue is in-progress.